packages/bpk-foundations-android/tokens/base.raw.android.json

```diff
@@ -2031,48 +2297,90 @@
       "originalValue": "{!LINE_HEIGHT_LG}",
       "name": "TEXT_LG_LINE_HEIGHT"
     },
+    "TEXT_LINK_DAY": {
+      "type": "color",
+      "category": "text-colors",
+      "value": "#0770e3ff",
+      "originalValue": "{!SKY_BLUE}",
+      "name": "TEXT_LINK_DAY"
+    },
+    "TEXT_LINK_NIGHT": {
+      "type": "color",
+      "category": "text-colors",
+      "value": "#6d9febff",
+      "originalValue": "{!NIGHT_BLUE}",
+      "name": "TEXT_LINK_NIGHT"
+    },
     "TEXT_PRIMARY_DARK_COLOR": {
-      "value": "#ffffffff",
       "type": "color",
       "category": "text-colors",
+      "value": "#ffffffff",
       "originalValue": "{!WHITE}",
       "name": "TEXT_PRIMARY_DARK_COLOR"
     },
-    "TEXT_PRIMARY_LIGHT_COLOR": {
+    "TEXT_PRIMARY_DAY": {
```

Comment:
here the old & new token will clash when it comes to the generated semantic token as both `TEXT_PRIMARY_DAY/NIGHT` and `TEXT_PRIMARY_LIGHT/DARK_COLOR` will result in a `textPrimary` token